python_modules/dagster/dagster/_core/ext/context.py

```diff
@@ -108,6 +119,7 @@ class ExtOrchestrationContext:
     message_handler: ExtMessageHandler
     context_injector_params: ExtParams
     message_reader_params: ExtParams
+    is_task_finished: bool = False
```

Comment:
nit: consider a name that more explicitly maps to what were tracking versus what we believe that corresponds to ie `has_exited_cm` something in that direction

Reply:
Hmm-- something like `has_exited_cm` feels very opaque to me.

Feels like we need a term for what happens in the scope of an `ext_protocol` block-- maybe an "ext session"? Then it could be `is_ext_session_closed`, meaning that any comms with the external process are terminated, which is what happens on `ext_protocol` exit.

Reply:
ext_session_closed sounds great to me